seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/job/JobResult.java

```diff
@@ -30,4 +30,8 @@ public class JobResult implements Serializable {
     @NonNull private JobStatus status;
 
     private String error;
+
+    public JobResult(@NonNull JobStatus status) {
```

Comment:
Where invoke this constructer?

Reply:
For external system integration use

e.g call throw exception